config/globals.js

```diff
@@ -9,52 +10,29 @@ const ENVIRONMENTS = {
 
 const { NODE_ENV = ENVIRONMENTS.DEVELOPMENT } = process.env;
 
-let defaultLogLevel = null;
-switch (NODE_ENV) {
-	case ENVIRONMENTS.PRODUCTION:
-		defaultLogLevel = 'error'; // level 3
-		break;
-	case ENVIRONMENTS.TEST:
-		defaultLogLevel = 'emerg'; // level 0
-		break;
-	case ENVIRONMENTS.DEVELOPMENT:
-	case ENVIRONMENTS.MIGRATION:
-	default:
-		defaultLogLevel = 'debug'; // level 7
-}
-
-let defaultDbUrl = null;
-switch (NODE_ENV) {
-	case ENVIRONMENTS.TEST:
-		defaultDbUrl = 'mongodb://127.0.0.1:27017/schulcloud-test';
-		break;
-	default:
-		defaultDbUrl = 'mongodb://127.0.0.1:27017/schulcloud';
-}
-
 const globals = {
 	BODYPARSER_JSON_LIMIT: process.env.BODYPARSER_JSON_LIMIT || '20mb',
-	DB_URL: process.env.DB_URL || defaultDbUrl,
-	DB_USERNAME: process.env.DB_USERNAME,
-	DB_PASSWORD: process.env.DB_PASSWORD,
+	DB_URL: Configuration.get('DB_URL'),
```

Comment:
Warum nicht gleich hier entfernen und die Stelle austauschen die das nutzen?

Reply:
that would be fine in legacy, however that portion of code gets used by new code as well, were you would need to call Configuration#get in multiple places, maybe not worth it for now

Reply:
ok